src/i18n.js

```diff
@@ -47,6 +47,7 @@ const languageList = {
     "ge": "ქართული",
     "uz": "O'zbek tili",
     "ga": "Gaeilge",
+    "sk-SK": "Slovenčina",
```

Comment:
We only have a translation file for sk: https://github.com/louislam/uptime-kuma/blob/507930e28a70e435a0ced07170360c17e2e10d80/src/lang/

And thanks for the large translation push ^^

```suggestion
    "sk": "Slovenčina",
```